Src/Base/AMReX_ParallelDescriptor.H

```diff
@@ -971,6 +971,7 @@ ParallelDescriptor::GatherLayoutDataToVector (const LayoutData<T>& sendbuf,
 
     int nprocs = ParallelContext::NProcsSub();
     Vector<int> recvcount(nprocs, 0);
+    recvbuf.resize(nprocs);
```

Comment:
The size is wrong.  I think it should be
```suggestion
    recvbuf.resize(sendbuf.size());
```

Reply:
Ah yes, number of boxes, not number of ranks. I've been mixing that up a lot lately with this graph stuff.

Fixed.